### PR TITLE
ibm5170_cdrom.xml: 13 New working software list additions

### PR DESCRIPTION
**Pull request by ArcadeShadow**
New working software list additions
--------------------------------------------
Humans 1 and 2 (Europe, OEM) [redump.org]
Humans 1 and 2 (Europe, re-release) [redump.org]
Prince of Persia 2 - The Shadow and the Flame (USA) [redump.org]
Prince of Persia CD Collection (Europe) [redump.org]
Prince of Persia CD Collection (USA) [redump.org]
Prince of Persia Collection: Limited Edition (Europe) [redump.org]
Prince of Persia Collection: Limited Edition (Europe, alt) [redump.org]
Prince of Persia Collection: Limited Edition (USA) [redump.org]
Prince of Persia Collection: Limited Edition (Netherlands) [redump.org]
Prince of Persia Collection: Limited Edition (Brazil) [redump.org]
Telstar 10 CD ROM Box Set [redump.org]
The Humans (Europe) [redump.org]
The Humans (USA) [redump.org]

### CODE REVIEW

**Comment by Allofich** [comment]
Just a minor thing, but for the title of Prince of Persia 2, in `ibm5170.xml` it is written as `Prince of Persia 2 - The Shadow &amp; the Flame`, using an ampersand.

I checked the game's page on MobyGames, and it looks like "and" is used for at least one in-game title screen
https://www.mobygames.com/game/macintosh/prince-of-persia-2-the-shadow-the-flame/screenshots/gameShotId,368887/ (from the Macintosh version but I assume it would be the same as the PC version)
The box and floppy disk, though, use "&" https://www.mobygames.com/game/prince-of-persia-2-the-shadow-the-flame/cover-art/gameCoverId,173203/, https://www.mobygames.com/game/prince-of-persia-2-the-shadow-the-flame/cover-art/gameCoverId,51012/.
Any rule of thumb for MAME when what is used when packaging and in-game titles differ?

File: hash/ibm5170_cdrom.xml [hunk 1]
```diff
@@ -3116,6 +3196,233 @@ https://www.pcgamingwiki.com/wiki/Myst
...
+Games:
+Prince of Persia (v1.4)
+Prince of Persia 2: The Shadow and the Flame (v1.1)
+Playable Demos:
+The Journeyman Project 3: Legacy of Time
+Riven: The Sequel to Myst
+Warlords III: Darklords Rising
+Videos:
+The Making of Prince of Persia 3D
+The Journeyman Project 3: Legacy of Time (included in demo)
```
Comment (Allofich) [inline hash/ibm5170_cdrom.xml:3356]: Does this mean there is also a video titled "The Journeyman Project 3: Legacy of Time" (same title as the game) on the CD?

**Review by angelosa — APPROVED** [review]
Not a fan of notes field starting with "CD contents" (it's obvious by the context) but meh I'll deal with it later.

File: hash/ibm5170_cdrom.xml (hunk 1 above)
Comment (ArcadeShadow) [reply hash/ibm5170_cdrom.xml]: updated this information confirming on the CD (there is no video).

**Comment by ArcadeShadow** [comment]
> Not a fan of notes field starting with "CD contents" (it's obvious by the context) but meh I'll deal with it later.

Agree. I removed this "CD content" indication.

**Comment by ICEknigh7** [comment]
Just noticed that the URL for "10 Game CD ROM" should be "http://redump.org/disc/34461/" instead of "http://redump.org/disc/51015/".

**Comment by Allofich** [comment]
There is currently a mix of "Windows95" and "Windows 95" for "platform".

The `ibm5170_cdrom.xml` file in the master branch has "Windows 95" for the one usage there (for id Anthology (rev 1)).

**Comment by ArcadeShadow** [comment]
> Just noticed that the URL for "10 Game CD ROM" should be "http://redump.org/disc/34461/" instead of "http://redump.org/disc/51015/".

Corrected the "copy/paste" error.

**Comment by ArcadeShadow** [comment]
> There is currently a mix of "Windows95" and "Windows 95" for "platform".
> 
> The `ibm5170_cdrom.xml` file in the master branch has "Windows 95" for the one usage there (for id Anthology (rev 1)).

Normalized the "Windows 95" platform value.

**Pull request merged**